sources/whippet/src/policy.rs

```diff
@@ -466,6 +466,118 @@ fn find_priority_threshold(rules: &[Rule]) -> u64 {
 }
 
 impl Rule {
+    /// Returns the "interface" value used in the final policy, replacing wildcards with empty
+    /// strings
+    /// See: https://github.com/bus1/dbus-broker/blob/e3324b3736fd40d95e7943fca6e485013d15d643/src/launch/policy.c#L97
+    pub(crate) fn interface(&self) -> Result<String> {
+        match self {
+            Self::Send { send_interface, .. } => {
+                if send_interface == "*" {
+                    Ok("".to_string())
+                } else {
+                    Ok(send_interface.to_owned())
+                }
```

Comment:
Either a macro or a function to abstract this logic would be great, there's a lot of repetition.